arm/Microsoft.Automation/automationAccounts/deploy.bicep

```diff
@@ -52,6 +55,13 @@ param eventHubAuthorizationRuleId string = ''
 @description('Optional. Name of the event hub within the namespace to which logs are streamed. Without this, an event hub is created for each log category.')
 param eventHubName string = ''
 
+@description('Optional. Type of managed service identity.')
```

Comment:
for managed identities I'd suggest to keep the same approach used by other modules for consistency (e.g. storage account)

Reply:
testing this

Reply:
Changed around on the approach so that it makes a bit more sense, and aligns more with how it is done in the portal experience.

Reply:
What do you think?